Added comment about hook files and initialization (#6756)
sveltejs/kit: documentation/docs/07-hooks.md

- `src/hooks.server.js` — your app's server hooks
- `src/hooks.client.js` — your app's client hooks

> Those files can also be used to run code at the start of the application. This can be useful for running initialization code for database clients or various other libraries.

Reviewer: ```suggestion
Code in these modules will run when the application starts up, making them useful for initializing database clients and so on.
```

> Those files can also be used to run code that runs at the start of the application. This can be useful for running initialization code for database ORMs or various other libraries.

Reviewer 2: If we want to add a comment like this I'd probably make it a bit more general to database clients in general and not just ORMs
```suggestion
Those files can also be used to run code that runs at the start of the application. This can be useful for running initialization code for database clients or various other libraries.
```
